examples/cockroachdb/cockroachdb-statefulset.yaml

```diff
@@ -163,11 +163,10 @@ spec:
   volumeClaimTemplates:
   - metadata:
       name: datadir
-      annotations:
-        volume.alpha.kubernetes.io/storage-class: anything
     spec:
       accessModes:
         - "ReadWriteOnce"
       resources:
         requests:
           storage: 1Gi
+      storageClassName: anything
```

Comment:
The alpha annotation has different behavior than the beta annotation.  The alpha annotation asks for a "default" storage class, which is installed on each major provider.  In GA, you request a default storageclass by not setting the storageClassName field, so this line should be removed if you want the example to use the default storageclass.  Otherwise, I would suggest changing the name to something else like "fast"